src/open_dread_rando/dread_patcher.py

```diff
@@ -174,7 +174,7 @@ def patch_spawn_points(editor: PatcherEditor, spawn_config: list[dict]):
         scenario = editor.get_scenario(scenario_name)
 
         editor.copy_actor(scenario_name, new_spawn_pos, base_actor, new_actor_name)
-        scenario.add_actor_to_entity_groups(collision_camera_name, new_actor_name)
+        scenario.add_actor_to_actor_groups(collision_camera_name, new_actor_name)
```

Comment:
I'm gonna just add `eg_` here to avoid making a breaking change.

Weirdly, the advanced patcher json worked perfectly fine when I exported and played it, so I think spawn points don't actually need to be in an actor group to function? Doesn't matter, but good to note.